doc/configuration.rst

```diff
@@ -1865,6 +1866,44 @@ each subsection, and a specific toctree for each subsection.
 In particular, sidebars generated using these toctrees might not reflect the
 actual section / folder structure.
 
+.. _manual_passthrough:
+
+Manually passing files
+======================
+
+By default, Sphinx-Gallery creates all the files that are written in the
+sphinx-build directory, either by generating rst and images from a ``*.py``
+in the gallery-source, or from  creating ``index.rst`` from ``README.txt``
+in the gallery-source.  However, sometimes it is desirable to pass files
+from the gallery-source to the sphinx-build.  For example, you may want
+to pass an image that a gallery refers to, but does not generate itself.
+You may also want to pass raw rst from the gallery-source to the
+sphinx-build, because that material fits in thematically with your gallery,
+but is easier to write as rst.  To accomodate this, you may set
+``copyfile_regex`` in ``sphinx_gallery_conf``.  The following copies
+across rst files.
+
+.. code-block:: python
+
+    sphinx_gallery_conf = {
+        ...
+       'copyfile_regex': r'.*\.rst',
+    }
+
+Note that if you copy across files rst files, for instance, it is your
+responsibility to ensure that they are in a sphinx ``toctree`` somewhere
+in your document.  You can, of course, add a ``toctree`` to your
+``README.txt``.
+
+Manually passing ``index.rst``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+If you want control over the gallery (or subgallery) ``index.rst``, you
+can bypass Sphinx-Gallery creating it from ``README.txt`` by including
+it in the gallery-source instead of the README.  If you do this, you are
```

Comment:
```suggestion
it in the gallery-source instead of the README and setting the `copyfile_regex` to a value that causes it to be copied, like ``r'index\.rst'`` or something more general.  If you do this, you are
```
Right?

Reply:
If I read something twice, I need to rewrite it ;-)

```
You can bypass Sphinx-Gallery automatically creating an index ``index.rst``
from a ``README.txt`` in a gallery directory or subdirectory. If your
``copyfile_regex`` includes ``index.rst``, and you have an ``index.rst`` in the
gallery-source instead of the README, Sphinx-Gallery will use that instead of
the index it automatically makes.  If you do this, you are responsible for
adding your own Sphinx ``toctree`` in that index (or elsewhere in your Sphinx
documentation) that includes any gallery items or other files in that
directory.
```

Reply:
Yeah my version was not great, just wanted to get the idea across :)